### DISCUSSION TITLE
ADC with DMA sample

### DISCUSSION THREAD

**Discussion by VinayShirol-eaton** [zephyrproject-rtos/zephyr#55080] [General]
HI , 
       I  am working on STM32u575zi_q board. do we have any sample which used ADC with DMA.

**Comment by erwango** [comment]
@VinayShirol-eaton DMA + ADC is still under review: https://github.com/zephyrproject-rtos/zephyr/pull/52965
Also H7 only for now, but U5 support should not require much work

**Reply by VinayShirol-eaton** [reply]
thanks @erwango , I will look into it.
